case-lib/opt.sh

```diff
@@ -146,5 +163,15 @@ func_opt_parse_option()
         } >> "$LOG_ROOT/version.txt"
     fi
 
-    unset _func_create_tmpbash _func_opt_dump_help _func_case_dump_description
+    unset _fill_opt_vars _dump_help_and_exit _dump_case_description
+}
+
+# Define common options among test scripts.
+add_common_options()
+{
+    # The help option
+    OPT_NAME['h']='help'
+    OPT_HAS_ARG['h']=0
+    OPT_VAL['h']=0
+    OPT_DESC['h']='show help information'
```

Comment:
Can you move this function back to the top of the file? It's a great example so a great way to start the file.